TodoList.UnitTests/Services/EmailSenderTest.cs

```diff
@@ -15,17 +16,19 @@ public class EmailSenderTest
         private readonly EmailSender _emailSenderService;
         private readonly Mock<ISendGridClient> _sendGridClientMock;
         private readonly Mock<HttpContent> _httpContentMock;
+        private readonly Mock<ILogger<EmailSender>> _logger;
 
         public EmailSenderTest()
         {
             _sendGridClientMock = new Mock<ISendGridClient>();
             _httpContentMock = new Mock<HttpContent>();
+            _logger = new Mock<ILogger<EmailSender>>();
```

Comment:
FYI: if there's no 'MoQ' functionality used, you can also use `new NullLogger<EmailSender>()` instead of mocking. If so, the comment above is not relevant anymore :)

Reply:
Nice suggestion, this change is done